index.html

```diff
@@ -4,9 +4,47 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/index.css">
+    <script src="https://kit.fontawesome.com/e77df31685.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
     <title>Weather Report</title>
 </head>
 <body>
-    
+    <div class="container">
```

Comment:
I know you will hear alot of back and forth about using divs and when to try to use semantic HTML. I wanted to share this guide on the flow for HTML. This will help decide what tags to use and when to use them:
![html_flowchart](https://user-images.githubusercontent.com/60710670/174199506-0428a0e7-59eb-41e9-b7ba-ce5db8f7d328.png)

Reply:
This is really helpful, thanks! I see div all the way at the end 😆